_blog/articles/2024-01-13-cloud-service-categories.md

```diff
@@ -0,0 +1,202 @@
+            </li>
+        </ul>
+    </li>
+</ul>
+
+<ul>
+    <li>Cloud provider is responsible for:
+        <ul>
+            <li>Hardware, Networking & Virtualization</li>
+        </ul>
+    </li>
+</ul>
+
+<ul>
+    <li>You are responsible for:
+        <ul>
+            <li>OS upgrades and patches</li>
+            <li>Application code, data and runtime</li>
+            <li>Configuring load balancing</li>
+            <li>Auto scaling</li>
+            <li>Availability</li>
+            <li>etc.. (and a lot of things!)</li>
+        </ul>
+    </li>
+</ul>
+
+</details>
+
+### PAAS (Platform as a Service)
+
+![Image](/images/cloud-service-categories/PAAS.gif "PAAS (Platform as a Service)")
+
+<details>
+    <summary>Notes</summary>
+    <ul>
+    <li>Use a platform provided by the cloud
+        <ul>
+            <li>Cloud provider is responsible for:
+                <ul>
+                    <li>Hardware, Networking & Virtualization</li>
+                    <li>OS (incl. upgrades and patches)</li>
+                    <li>Application Runtime</li>
+                    <li>Auto scaling, Availability & Load balancing etc..</li>
+                </ul>
+            </li>
+            <li>You are responsible for:
+                <ul>
+                    <li>Configuration (of Application and Services)</li>
+                    <li>Application code/data (if needed)</li>
+                </ul>
+            </li>
+        </ul>
+    </li>
+</ul>
+
+</details>
+
+### SaaS (Software as a Service)
+
+![Image](/images/cloud-service-categories/SAAS-08-12-2023.gif "SaaS (Software as a Service)")
+
+<details>
+    <summary>Notes</summary>
+    <ul>
+    <li>Centrally hosted software (mostly on the cloud)
+        <ul>
+            <li>Offered on a subscription basis (pay-as-you-go)
+                <ul>
+                    <li>Examples:
+                        <ul>
+                            <li>Email, calendaring & office tools (such as Outlook 365, Microsoft Office 365, Gmail, Google Docs)</li>
+                            <li>Customer relationship management (CRM), enterprise resource planning (ERP), and document management tools</li>
+                        </ul>
+                    </li>
+                </ul>
+            </li>
+            <li>Cloud provider is responsible for:
+                <ul>
+                    <li>OS (incl. upgrades and patches)</li>
+                    <li>Application Runtime</li>
+                    <li>Auto scaling, Availability & Load balancing etc..</li>
+                    <li>Application code and/or Application Configuration (How much memory? How many instances? ..)</li>
+                </ul>
+            </li>
+            <li>Customer is responsible for:
+                <ul>
+                    <li>Configuring the software!</li>
+                    <li>Your data</li>
+                </ul>
+            </li>
+        </ul>
+    </li>
+</ul>
+
+</details>
+
+### Example 1: VMs with Load Balance
```

Comment:
Example 1: Deploying VMs with a Load Balancer

Do not change image (only heading in markdown)